UPGRADING

```diff
@@ -467,6 +467,12 @@ PHP 8.2 UPGRADE NOTES
   Previously only hexadecimal prefixes and using a leading 0 for octal numbers
   was supported.
 
+- Parsing of invalid values will now trigger a warning when this was silently
+  ignored before. Interpretation of these values is not changed, for
+  backwards compatibility. Testing configuration can fail though.
+  This affects the following settings:
+  . {fpm_pool}.listen
+
```

Comment:
I see some conflicts as this is getting updated by other changes. Feel free to remove it and just leave the text in the comment. It can be added later during merge in the same way like we update NEWS...